guides/common/modules/proc_changing-the-repository-sets-status-in-project.adoc

```diff
@@ -0,0 +1,18 @@
+[id="Changing_the_Repository_Sets_Status_in_{project-context}_{context}"]
+= Changing the Repository Sets Status in {Project}
+
+Repository sets show available repositories to each host.
+If your host is subscribed to a particular Content View and Lifecycle environment, the default repository set is limited to the repositories that are specifically available to that host.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts* and select a host.
+. Select the *Repository sets* tab.
+On the tab, there is a set of repositories available to each host with a status of *Enabled* or *Disabled*.
+. You can override the default status by using the action menus on each table row by changing the status to *Override to disabled*, *Override to enabled*, or *Reset to default*.
```

Comment:
You can also bulk-select with the checkboxes on each table row and use the action menu at the top.